src/test/java/com/google/api/codegen/testdata/ruby_doc_version_index_library.baseline

```diff
@@ -50,6 +50,20 @@
 # [Product Documentation]: https://cloud.google.com/library
 #
 module Library
+  VERSION_MATCHER = /
```

Comment:
Can we put this regex in GAX instead of codegen? It will make updating much easier if we ever need to loosen it.